rules/windows/execution_pdf_written_file.toml

```diff
@@ -0,0 +1,104 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a suspicious file that was written by a PDF reader application and subsequently executed. These processes are
+often launched via exploitation of PDF applications.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution of File Written or Modified by PDF Reader"
+risk_score = 21
+rule_id = "1defdd62-cd8d-426e-a246-81a37751bb2b"
+severity = "high"
+tags = ["from-eql-rules"]
+type = "eql"
+
+query = '''
+sequence with maxspan=2h
+  [file where event.type != "delete" and file.extension == "exe" and
+     process.name in ("acrord32.exe", "rdrcef.exe", "foxitphantomPDF.exe", "foxitreader.exe") and
+     not file.name in ("foxitphantomPDF.exe",
```

Comment:
```suggestion
     file.name not in ("foxitphantomPDF.exe",
```

Reply:
what do you perceive the FN risk to be for this `not in (...)` check?
could be worth looking into some reports to see if any of these names have been used to blend in

Reply:
or alternatively remove altogether with the suggestion to tune based on environment